traitsui/testing/tester/wx/implementation/enum_editor.py

```diff
@@ -0,0 +1,123 @@
+from traitsui.testing.tester.base_classes import _SourceWithLocation
+from traitsui.testing.tester.wx import helpers
+
+
+class _IndexedListEditor(_SourceWithLocation):
+    """ Wrapper class for EnumListEditor and Index.
+    """
+    source_class = ListEditor
+    locator_class = locator.Index
+    handlers = [
+        (command.MouseClick, (lambda wrapper, _: helpers.mouse_click_listbox(
+            control=wrapper.target.source.control,
+            index=wrapper.target.location,
+            delay=wrapper.delay))
+        )
+    ]
+
+
+class _IndexedRadioEditor(_SourceWithLocation):
+    """ Wrapper class for EnumRadioEditor and Index.
+    """
+    source_class = RadioEditor
+    locator_class = locator.Index
+    handlers = [
+        (command.MouseClick, (lambda wrapper, _: helpers.mouse_click_child_in_panel(
+                control=wrapper.target.source.control,
+                index=wrapper.target.location,
+                delay=wrapper.delay))
+        )
+    ]
+
+
+class _IndexedSimpleEditor(_SourceWithLocation):
+    """ Wrapper class for Simple EnumEditor and Index.
+    """
+    source_class = SimpleEditor
+    locator_class = locator.Index
+    handlers = [
+        (command.MouseClick, (lambda wrapper, _: helpers.mouse_click_combobox_or_choice(
+                control=wrapper.target.source.control,
+                index=wrapper.target.location,
+                delay=wrapper.delay))
+        ),
+    ]
+
+
+def displayed_text_handler(wrapper, interaction):
+    """ Handler function used to query DisplayedText for different
+    styles of Enum Editor. 
+
+    Parameters
+    ----------
+    wrapper : UIWrapper
+        The UIWrapper containing that object with text to be displayed. 
+    interaction : query.DisplayedText
+        Unused in this function but included to match the expected format of a
+        handler.  Should only be query.DisplayedText
+    """
+    control = wrapper.target.control
+    if isinstance(control, wx.ComboBox):
+        return control.GetValue()
+    else: # wx.Choice or wx.ListBox
+        return control.GetString(control.GetSelection())
+
+
+def register(registry):
+    """ Registry location and interaction handlers for EnumEditor.
+
+    Parameters
+    ----------
+    registry : InteractionRegistry
+    """
+    _IndexedListEditor.register(registry)
+    _IndexedRadioEditor.register(registry)
+    _IndexedSimpleEditor.register(registry)
+
+
+    simple_editor_text_handlers = [
+        (command.KeyClick, (lambda wrapper, interaction:
+            helpers.key_click_text_ctrl(
+                control=wrapper.target.control,
+                interaction=interaction,
+                delay=wrapper.delay))
+        ),
+        (command.KeySequence, (lambda wrapper, interaction:
+            helpers.key_sequence_text_ctrl(
+                control=wrapper.target.control,
+                interaction=interaction,
+                delay=wrapper.delay))
+        ),
+    ]
```

Comment:
`key_click_text_ctrl` and `key_sequence_text_ctrl` are actually not at all `wxTextCtrl` specific any more after the removal of the `key_click` function and the use of `EmulateKeyPress`.  The only methods they use are just specific to `wxControl` and `wxTextEntry`.  In this case, `wvComboBox` subclasses both (just as `wxTextCtrl` does).  
Thus I think `key_click_text_ctrl` and `key_sequence_text_ctrl` can probably be renamed.  However, we may ultimately want a keyclick and key sequence function specific to text controls if we do ever get the old key_click function working which would allow things like '\b' to be typed, etc.

Reply:
Any changes related to this could probably go into a separate PR